torch/quantization/quantize_fx.py

```diff
@@ -115,16 +115,34 @@ def quantize_static_fx(model, qconfig_dict, run_fn, run_args, inplace=False, deb
 
     Args:
         `model`: input float TorchScript model
-        `qconfig_dict`: qconfig_dict is a dictionary with names of sub modules as key and
-        qconfig for that module as value, empty key means the qconfig will be applied
-        to whole model unless it’s overwritten by more specific configurations, the
-        qconfig for each module is either found in the dictionary or fallback to
-         the qconfig of parent module.
-
-        Right now qconfig_dict is the only way to configure how the model is quantized,
-        and it is done in the granularity of module, that is, we only support one type
-        of qconfig for each torch.nn.Module, and the qconfig for sub module will
-        override the qconfig for parent module, empty string means global configuration.
+        `qconfig_dict`: qconfig_dict is a dictionary with the following configurations:
+    qconfig_dict = {
+    # optional, global config
+    "": qconfig?,
+
+    # optional, used for module and function types
+    # could also be split into module_types and function_types if we prefer
+    "object_type": [
+      (nn.Conv2d, qconfig?),
+      (F.add, qconfig?),
```

Comment:
sorry, didn't know these would make it to the code :) maybe we can use `torch.nn` and `torch.nn.functional`?